src/Ledger/Conway/Specification/Epoch.lagda

```diff
@@ -85,13 +85,20 @@ instance
   HasPParams-EpochState .PParamsOf = PParamsOf ∘ EnactStateOf
 \end{code}
 \begin{NoConway}
+
+`PoolDistr`{.AgdaBound} in the Shelley spec results from dividing the coins in
+`PoolDelegatedStake`{.AgdaBound} by the total stake in
+`PoolDelegatedStake`{.AgdaBound}.
```

Comment:
- Since this file is still latex, I'd suggest to use latex markup so the pipeline doesn't get confused.
```suggestion
\AgdaBound{PoolDistr} in the Shelley spec results from dividing the coins in
\AgdaBound{PoolDelegatedStake} by the total stake in
\AgdaBound{PoolDelegatedStake}.
```
- I'd add a reference to the definition of PoolDistr in the Shelley spec
- If I understand correctly, this paragraph aims to differentiate the type PoolDelegatedStake introduced here from the type used in Shelley spec (PoolDistr). I suggest to expand on the explanation, because without context is hard to understand.